trieste/models/gpflux/architectures.py

```diff
@@ -0,0 +1,175 @@
+
+#
+# Copyright (c) 2021 The GPflux Contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This module provides :func:`build_constant_input_dim_deep_gp` to build a Deep GP of
+arbitrary depth where each hidden layer has the same input dimensionality as the data.
+"""
```

Comment:
I assume you accidentally put this file under version control here, is it a complete copy-paste from gpflux?

Reply:
ohh sorry..this was a temporary easy fix so I can write my units tests faster